Week-4/OOP_js/prototype&Object/script.js

```diff
@@ -0,0 +1,61 @@
+//Protecting the Object
+const person = {
+  name: 'Myname',
+  age: 24,
+  email: 'ab@hopmail.com',
+};
+Object.defineProperty(person, 'name', {
+  value: 'Myname',
+  writable: false,
+});
```

Comment:
this is simply to prevent write on name field?

Reply:
Yes to make it read-only